client/components/LearnerViewDirective.js

```diff
@@ -205,6 +216,20 @@ tie.directive('learnerView', [function() {
         .tie-dot-3 {
           -webkit-animation-delay: 0.2s;
         }
+        .tie-question-ui-inner {
+          padding-left: 32px;
+          padding-right: 32px;
+          white-space: nowrap;
+        }
+        .tie-question-ui-outer {
+          display: table;
+          margin-left: auto;
+          margin-right: auto;
+          margin-top: 16px;
+        }
```

Comment:
Why are the above two statements (-inner and -outer) here? It looks like it was copy/pasted from below.